### PR TITLE
GraFx Developer Remodel for Training

### PR DESCRIPTION
**Pull request by seancrowe**
This is a big remodel, which includes many different training materials.

This is still a WIP. You may find broken links and unfinished pages. Content will change over time.

### CODE REVIEW

File: docs/GraFx-Developers/grafx-studio/environment-api-quickstart/02-managing-integrations/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Managing Integration
```
Comment (brapoprod) [inline docs/GraFx-Developers/grafx-studio/environment-api-quickstart/02-managing-integrations/index.md:1]: Where is part 01?

Comment (BramVerniest) [reply docs/GraFx-Developers/grafx-studio/environment-api-quickstart/02-managing-integrations/index.md:1]: Sean told me, it wasn't ready yet. Will be in a future update.

**Comment by seancrowe** [comment]
@BramVerniest this is everything to be ready for a draft as there may be some misspellings, broken links, or even content that needs to be rearranged or modified.

My suggestion is that you do a quick review of this PR, making sure there are no glaring issues, and then we ask Niels, Brandon, Brian, yourself, our employees, and whomever to review and correct things as we go.

The other approach is that this PR will stay as a PR for weeks as we fine tune everything.

**Review by BramVerniest — APPROVED** [review]
What another great addition
LGTM

**Pull request merged**